### DISCUSSION TITLE
Fetch API or library like axios

### DISCUSSION THREAD

**Discussion by n-igarashi** [monstar-lab-oss/ml-frontend#7] [General]
Using fetch or library? axios, SWR, etc.
https://github.com/vercel/swr
https://github.com/Kong/swrv

**Comment by don-wang** [comment]
Differences between Axios and Fetch: 
![image](https://user-images.githubusercontent.com/1377797/142787453-989c102e-f18f-454e-96ba-6016afab9599.png)

**Reply by don-wang** [reply]
It makes simpler by using fetch, but axios provides more function and customize possibility
